udata/harvest/backends/base.py

```diff
@@ -84,11 +84,8 @@ def perform_initialization(self):
             if not self.dryrun:
                 self.job.save()
         except Exception as e:
-            log.error("Error in initialization : %s" % (str(e)))
-            log.exception(e)
             self.job.status = 'failed'
-            error = HarvestError(message=str(e),
-                                 details=traceback.format_exc())
```

Comment:
Yes, as a developer, but regular user are scared and not helped by this stacktrace.
You have a far more detailed version in Sentry or in the shell while developping.

Idea of improvement in another PR: maybe get the Sentry ID